extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/InitTaskProcessor.java

```diff
@@ -20,7 +20,7 @@
 
 public class InitTaskProcessor {
 
-    private static final String INIT_CONTAINER_WAITER_NAME = "init";
+    private static final String INIT_CONTAINER_WAITER_NAME_PREFIX = "wait-for-";
     private static final String INIT_CONTAINER_WAITER_DEFAULT_IMAGE = "groundnuty/k8s-wait-for:no-root-v1.7";
```

Comment:
unrelated to that PR - we must make this image configurable.

Reply:
AFAIR, the image is configurable using the standard k8s properties for setting images.
I think that `quarkus.kubernetes.init-containers.wait-for-<init task name>.image=<my image> should work.

Reply:
We need a better image, that one is crippled with CVEs.

I didn't find a test checking that we can change the image.